core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -153,13 +272,13 @@ <h2 ng-if="collectionPlaythrough.hasFinishedCollection()" class="oppia-page-head
              <circle ng-show="($index===activeHighlightedIconIndex)?true:false"
                      cx="50"
                      cy="70"
-                     r="45"
+                     r="50"
                      id="<['highlight' + $index]>"
-                     fill="none"
-                     stroke="#A6DACF"
-                     stroke-width="10"
-                     stroke-opacity="0.5"/>
-             <circle cx="50"
+                     fill="#A6DACF"
+                     fill-opacity="0.5"
+                     />
+             <circle ng-class="(isNextExploration(node.getExplorationId()))?'animate-next-exploration':''"
```

Comment:
Suggest using the {...} notation for ng-class. It's clearer.

ng-class={'animate-next-exploration': isNextExploration(node.getExplorationId())}